addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/dashboard/Dashboard.js

```diff
@@ -218,20 +243,26 @@ class Dashboard extends React.Component {
   }
 
   renderPastMonthCostPerEnv() {
-    if (_.isEmpty(this.state.envNameToCostInfo)) {
+    if (_.isEmpty(this.state.envIdToCostInfo)) {
       return <ProgressPlaceHolder />;
     }
 
     const pastMonthCostTotalArray = [];
-    Object.keys(this.state.envNameToCostInfo).forEach(envName => {
+    Object.keys(this.state.envIdToCostInfo).forEach(envId => {
       let total = 0;
-      Object.keys(this.state.envNameToCostInfo[envName].pastMonthCostByUser).forEach(user => {
-        total += this.state.envNameToCostInfo[envName].pastMonthCostByUser[user];
+      Object.keys(this.state.envIdToCostInfo[envId].pastMonthCostByUser).forEach(user => {
```

Comment:
Would `Object.values(this.state.envIdToCostInfo[envId].pastMonthCostByUser).reduce((a, b) => a + b)` work?

Reply:
Yeah, that works too. Will replace it